variables

```diff
@@ -12,3 +12,4 @@ TELEGRAM_CHAT_ID=-123456
 VALIDATOR_EJECTOR_LOCATOR_ADDRESS=0x1eDf09b5023DC86737b59dE68a8130De878984f5 # Goerli contract. No mainnet contract exists currently.
 VALIDATOR_EJECTOR_STAKING_MODULE_ID=0
 VALIDATOR_EJECTOR_OPERATOR_ID=0
+VALIDATOR_EJECTOR__MESSAGES_PASSWORD=pass
```

Comment:
Could you replace the double underscore on the single? To match the rest style of the variables writing.
VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD

Reply:
> Could you replace the double underscore on the single? To match the rest style of the variables writing. VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD

got it! sorry